book/src/example/clone/csi-clone.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
are these examples referenced anywhere?

Reply:
They were not, I'll add an Example header in the cloning doc and point to them there.